### PR TITLE
CF-79 Verify migration of floating IPs into pre-created networks.

### PR DESCRIPTION
**Pull request by plomakin**
Fixed generate_load.
Added dst external network.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:27: [W0403(relative-import), ] Relative import 'base', should be 'tests.base'
devlab/tests/generate_load.py:28: [W0403(relative-import), ] Relative import 'cleanup', should be 'tests.cleanup'
devlab/tests/generate_load.py:29: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:594: [E1101(no-member), Prerequisites.create_security_group] Instance of 'dict' has no 'id' member
devlab/tests/generate_load.py:668: [E1101(no-member), Prerequisites.create_cinder_volumes] Instance of 'dict' has no 'id' member

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: devlab/tests/config.py [hunk 1]
```diff
@@ -28,6 +28,10 @@
 filters_file_naming_template = 'filter_{tenant_name}.yaml'
 """Path to store filter files"""
 
+ext_net_map = 'ext_net_map.yaml'
```
Comment (roman-verchikov) [inline devlab/tests/config.py]: Looks like you forgot to add `ext_net_map.yaml` file to commit

Comment (plomakin) [inline devlab/tests/config.py]: nope, it's generating by generate_load.py

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:27: [W0403(relative-import), ] Relative import 'base', should be 'tests.base'
devlab/tests/generate_load.py:28: [W0403(relative-import), ] Relative import 'cleanup', should be 'tests.cleanup'
devlab/tests/generate_load.py:29: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:598: [E1101(no-member), Prerequisites.create_security_group] Instance of 'dict' has no 'id' member
devlab/tests/generate_load.py:672: [E1101(no-member), Prerequisites.create_cinder_volumes] Instance of 'dict' has no 'id' member

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module tests.generate_load
devlab/tests/generate_load.py:27: [W0403(relative-import), ] Relative import 'base', should be 'tests.base'
devlab/tests/generate_load.py:28: [W0403(relative-import), ] Relative import 'cleanup', should be 'tests.cleanup'
devlab/tests/generate_load.py:29: [W0403(relative-import), ] Relative import 'config', should be 'tests.config'
devlab/tests/generate_load.py:598: [E1101(no-member), Prerequisites.create_security_group] Instance of 'dict' has no 'id' member
devlab/tests/generate_load.py:672: [E1101(no-member), Prerequisites.create_cinder_volumes] Instance of 'dict' has no 'id' member

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
I suspect tests require some more tuning:
```
Parameter provider:segmentation_id for resource networks with name mynetwork1 are different src: 1, dst: 7
    """Fail immediately, with the given message."""
>>  raise self.failureException('Parameter provider:segmentation_id for resource networks with name mynetwork1 are different src: 1, dst: 7')
```

**Comment by plomakin** [comment]
this one is incorrect

**Pull request closed without merging**